### PR TITLE
Add 3 to title

### PR DESCRIPTION
**Pull request by jrift**
we need to test the process.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ function App() {
     <div className="App">
       
       <header>
-        <h1>Fitness App</h1>
```
Comment (ElijahQuinones) [inline src/App.js:9]: Why are we adding 3 to Fitness app on the h1?

Comment (jrift) [reply src/App.js:9]: For testing

Comment (ElijahQuinones) [reply src/App.js:9]: sounds good

**Pull request merged**